Makefile

```diff
@@ -48,6 +48,12 @@ install: check-ledger
 install_debug:
 	go install $(BUILD_FLAGS) ./cmd/thorchaindebug
 
+recreate: check-ledger
+	rm -rf ~/.thorchain*
+	make install
+	thorchaind init --name local_validator --chain-id test-chain-local
+	thorchaind start
```

Comment:
That's a bit too destructive for my taste – it would for example delete all keys stored by `thorchaincli`. Could you quickly test whether `thorchaind unsafe_reset_all` serves your purpose?

Reply:
yeh, full delete is the intention - could name it differently to be more clear?

I don't think thorchaind unsafe_reset_all would redo the genesis process/new genesis file, but i may be mistaken?

Reply:
@musnit okay, could you restrict deletion in that case to just `~/.thorchaind`? I am just a bit concerns that cli data such as private keys are deleted by accident at some point in the future.